README.md

```diff
@@ -55,3 +62,15 @@ try {
 
 ```
 
+## Limitations
+
+- Non-ember npm dependencies are not supported.  It is assumed all resolutions one of:
+   - classic ember apps
+   - classic ember addons
+   - v2 ember addons
+
+- No support for app namespace merging: so files in an addon's `app` directory will not be resolved.
```

Comment:
This is "easy" to add support for, but would require more I/O so should be behind a flag that opts-in to spend I/O for this feature

Reply:
Can you either add another entry to this list or update this entry to make it clear that `<addon root>/test-support` is not supported at all (it is merged into `<app name here>/tests` akin to how an addon's `app/` folder is merged).